### ISSUE TITLE
Feature request: allow N jobs at once

### ISSUE THREAD

**Issue by snim2** [adapteva/epiphany-sdk#39]
Building a toolchain from scratch is very slow. It would be nice to have something like the `-j` option in Make to run several jobs in parallel. Things like cloning repositories could easily be done in parallel, even if the compile cycles cannot.

**Comment by olajep** [comment]
That is already happens automagically for building.
https://github.com/adapteva/epiphany-sdk/blob/2014.11/build-toolchain.sh#L699
You can also pass the --jobs flag to build-toolchain.sh

Don't think it's worth the hassle to clone in parallel. It should be mostly I/O bound anyways?

// Ola

**Comment by snim2** [comment]
Thanks, I didn't know about the `--jobs` switch.

The instructions all say to run `build-epiphany-sdk.sh` and that doesn't have a `--jobs` flag listed in usage:

```bash
$ ./sdk/build-epiphany-sdk.sh -h
Usage: ./build-epiphany-sdk.sh 
    [-a <host arch>]: The host architecture. Default armv7l
    [-c <host triplet>]: The architecture triplet. Only needed for Canadian cross builds.
    [-C]: Clean before start building.
    [-d]: Enable building with debug symbols.
    [-r <revision>]: The revision string for the SDK. Default RevUndefined
    [-R]: Do release build. Use define-release.sh to get right tags
    [-t <tag_name>]: The tag name (or branch name) for the SDK sources. Default master
    [-h]: Show usage
```
would this be worth adding (to be passed to `build-toolchain.sh`)?

**Comment by olajep** [comment]
Fixed here
https://github.com/adapteva/epiphany-sdk/commit/7f5897b5af7dc90801e60f3144d45fd906d1998d
